src/commands/showReferrer.ts

```diff
@@ -0,0 +1,86 @@
+import * as vscode from 'vscode';
+import { execAsync } from '../support/execAsync';
+import { window } from 'vscode';
+import * as os from 'os';
+import * as path from 'path';
+import { access } from 'fs/promises'; 
+
+const errorMessage = 'ORAS not found on default path "C:\\Users\\userName\\bin\\oras". Download ORAS or update path: ';
+const goToOrasButton = 'Download ORAS';
+const validTag = /^[a-z0-9./_:-]+$/; 
+
+/* checks that oras exists on user computer through building a default path and checking 
+ * the path exists. Path is the configured oras directory and bool states if the path exists (true) or doesn't (false).
+ */
+async function getOrasPath(path: any, bool : boolean): Promise<[string, boolean]> {
+    const isWindows = os.platform() === 'win32'; 
+    const homedir = os.homedir(); 
+    const orasExecutable = isWindows ? 'oras.exe' : 'oras'; 
+    const dirOras = path.join(homedir, 'bin', 'oras', orasExecutable);
```

Comment:
how do we know this is the correct install directory for linux? have we tried this?